ReportPortal.Client/Extention/HttpResponseMessageExtension.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace ReportPortal.Client.Extention
 {
-    static class HttpResponseMessageExtension
+    public static class HttpResponseMessageExtension
```

Comment:
I made it public, because the retried which we plan to move to Common solution references to this method.
I propose not to duplicate this extension in 2 solutions.
Also I suggest to add the retried to common together with this release 3.0.0, because currently it references the 2.1.9 version where this class is still internal.